src/views/MaintenancePage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <main class="maintenance-page">
+        <div class="maintenance-page_content level">
+            <img
+                class="maintenance-page_img"
+                src="../assets/img/mantenimiento-image.svg"
+                alt="Maintenance Image"
+                title="Maintenance-Image"
+            />
+            <div class="maintenance-page_description level">
+                <div class="maintenance-page_title">
```

Comment:
@gonzalo-plaza It's a good practice use h1 to titles and h2 to subtitles, cause the SEO works veri good with h1 and h2, in maintenance is not importatn but.... if good use this